samples/event_manager/src/events/control_event.c

```diff
@@ -17,7 +17,9 @@ EVENT_INFO_DEFINE(control_event,
 		  ENCODE(),
 		  profile_control_event);
 
+__attribute__((__used__)) const uint8_t control_event_flags = BIT(EVENT_FLAGS_INIT_LOG_ENABLE);
```

Comment:
Why `__used__`? We can (and even should) opt this out if possible - the value will be stored in the structure that defines event type.

Reply:
why not static?